src/lib/config_v2/config.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"sync"
+	"time"
+
+	"gopkg.in/yaml.v3"
+)
+
+//Config struct stores the APIs parsed from local-config.yml file.
+type Config struct {
+	path string
+	mux  sync.RWMutex
+	APIs []API `yaml:"apis"`
+}
+
+//API stores the information of the endpoints.
+type API struct {
+	Name    string            `yaml:"name,omitempty"`
+	Every   time.Duration     `yaml:"every,omitempty"`
+	Domain  string            `yaml:"domain_or_ip,omitempty"`
+	Route   string            `yaml:"route,omitempty"`
+	Method  string            `yaml:"method,omitempty"`
+	Headers map[string]string `yaml:"headers,omitempty"`
+	Params  map[string]string `yaml:"params,omitempty"`
+	Body    map[string]string `yaml:"body,omitempty"`
+}
+
+//Instantiate a Config object and reloads the data from the given file location.
+func NewConf(path string) (*Config, error) {
+	c := &Config{
+		path: path,
+	}
+	c, err := c.Reload()
+	if err != nil {
+		return nil, fmt.Errorf("creating config: %w", err)
+	}
+	return c, nil
+}
+
+//Reloads data from the config file.
```

Comment:
```suggestion
// Reload reloads data from the config file.
```
Remember go comment is written as

`// <object_name_exact> <comment>`